### PR TITLE
👷Fix iOS Build

### PR DESCRIPTION
**Pull request by JackDevAU**
- Updates Mac OS Virtual environment for iOS step from `10.15` to `11`
	- Adds workaround for macos-11 dependency install issue

### CODE REVIEW

**Review by JackDevAU — COMMENTED** [review]
Current Progress: 

Fixed iOS and Android builds :)

Notes:

macos-10.15 works for Android 
macos-10.15 works for iOS IF we set the Release Linker to `None` instead of `SDK`

If we do set to link `SDK` then we get the following build error:

```bash
"/Users/runner/work/SSW.Rewards/SSW.Rewards/Xamarin/SSW.Rewards/SSW.Rewards.iOS/SSW.Rewards.iOS.csproj" (default target) (1) ->
(_CompileToNative target) -> 
  MTOUCH : error MT2101: Can't resolve the reference 'System.Void UIKit.UITableView::set_SectionHeaderTopPadding(System.nfloat)', referenced from the method 'System.Void Xamarin.Forms.Platform.iOS.ListViewRenderer::OnElementChanged(Xamarin.Forms.Platform.iOS.ElementChangedEventArgs`1<Xamarin.Forms.ListView>)' in 'Xamarin.iOS, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065'. [/Users/runner/work/SSW.Rewards/SSW.Rewards/Xamarin/SSW.Rewards/SSW.Rewards.iOS/SSW.Rewards.iOS.csproj]
```

If we change iOS to macos-11:
- We can no longer install deps.... 
```bash
 NU1100: Unable to resolve 'Com.Airbnb.Xamarin.Forms.Lottie (= 3.0.3)' for 'Xamarin.iOS,Version=v1.0'.
```

[macOS 10.15 info](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md)

[macOS 11.6 info](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md)

File: .github/workflows/xamarin.yml [hunk 1]
```diff
@@ -62,11 +62,25 @@ jobs:
...
         fetch-depth: 0
+    - name: Set default VS version
+      run: |
+        mv "/Applications/Visual Studio.app" "/Applications/Visual Studio 2022.app"
+        mv "/Applications/Visual Studio 2019.app" "/Applications/Visual Studio.app"
+
+      # macOS agents recently have empty NuGet config files, resulting in restore failures,
+      # see https://github.com/actions/virtual-environments/issues/5768
+      # Add the global nuget package source manually for now.
+    - name: Setup NuGet.Config
```
Comment (wicksipedia) [inline .github/workflows/xamarin.yml:78]: Maybe copy the existing nuget file that's already in the repo instead of creating a new one?

**Review by wicksipedia — CHANGES_REQUESTED** [review]

File: .github/workflows/xamarin.yml (hunk 1 above)
Comment (JackDevAU) [reply .github/workflows/xamarin.yml]: done

**Review by william-liebenberg — APPROVED** [review]

**Review by wicksipedia — APPROVED** [review]
lgtm

**Pull request merged**